### ISSUE TITLE
issues with qt-5.6.2

### ISSUE THREAD

**Issue by Rodolphe21** [ContinuumIO/anaconda-issues#3581]
Dear Team,

I have successfully installed Anaconda2, installed numpy, scikit-learn, pandas etc. except that I have issues with the following:

- when installing matplotlib (conda install matplotlib) or updating the navigator (conda update anaconda-navigator) or when creating an environment for a project base on a *.yml file I get the following error message:

ERROR conda.core.link: _execute_actions(337): An error occurred while installing package 'defauls::qt-5.6.2-vc9.5', 
WindowsError(2, 'the system cannot find the file specified') etc.

In other words I am blocked in my project... because windows rolls back all it has installed after the error msg.

I have checked in c:\Users\...\Anaconda2\pkg and both "qt-5.6.2-vc14_3" and "qt-5.6.2-vc9_5" are available and have tried to hide the one or the other to see if it brings something, but it didn't.

conda info:
win64
conda & conda-env version 4.3.23
python version 2.7.13.final.0
request 2.14.2

> Can you please tell me what is wrong with the conda installer?

Thank you very much

Rodolphe

**Comment by ccordoba12** [comment]
Instead of

    defauls::qt-5.6.2-vc9.5

this shouldn't be

    defauls::qt-5.6.2-vc9_5

?

**Comment by Rodolphe21** [comment]
have checked again right now...

in the error message it is exactly: defaults::qt-5.6.2-vc9_5

with "underscore" betwen 9 and 5

...and in fact ">conda install qt" doesn't work for the same reason...

**Comment by ccordoba12** [comment]
So, what can do in our side? Is this a conda problem?

**Comment by mingwandroid** [comment]
No, it's likely a virus scanner that's removed a file.

**Comment by mingwandroid** [comment]
Can you try `conda update conda` before trying to update or install Qt?

**Comment by Rodolphe21** [comment]
Good point... I also thought about my antivirus blocking one file...

> conda update conda
and
> conda install qt
bring the same error

Do you have a list of all the files that are normally to be found in the pkg qt? 

For me to check...

**Comment by Rodolphe21** [comment]
so I have decided to reinstall anaconda taking the zip file (the advice for people behind firewalls), but run again in the same sort of problems... I am really frustrated, already lost 5 days on this...

1st issue ###
I do need to change the PATH env var by hand (....;C:\Users\rodolphe_masson\AppData\Local\Continuum\Anaconda2\Scripts), even if it is said that we don't need to do that with 4.4.0., otherwise conda is not recognized as a callable function

2nd issue ###
When I start the cmd window from the anaconda menu, I get the following error message right at the top of the window:
'@"C:\Users\rodolphe_masson\AppData\Local\Continuum\Anaconda2\Scripts\..\python.exe" -c "import ctypes; print(ctypes.cdll.kernel32.GetACP())" is not recognized as an internal or external command, operable program of batch file.
The filename, directory name, or volune labe syntax is incorrect.

3rd issue ###
conda info gives 4.3.23 (after a "conda update conda"),  whereas we have anaconda 4.4... am lost in the versioning...

4th issue ###
qt-6.5.2_9_5 still doesn't want to install and that blocks matplotlib and the creation of envs from .yml files... (no issue with Numpy, IPython, scikit-learn)
conda install qt => proposes to install 5.6.2-vc9_5 [vc9] and I answer with yes... it downloads it within 10 sec (5,45 Mb/s) but then brings the error message already reported:

ERROR conda.core.link: _execute_actions(337): An error occurred while installing package 'defauls::qt-5.6.2-vc9.5',
WindowsError(2, 'the system cannot find the file specified')
Attempting to roll back...

I would love to do "data science" instead of "windows/anaconda" admin... am so desesperate...

Help!?!

Thank you for any new idea

Rodolphe

**Comment by Rodolphe21** [comment]
additionally, 
> conda update qt
doesn't work: PackageNotInstalledError
I can only test with
> conda install qt

**Comment by mingwandroid** [comment]
> I do need to change the PATH env var by hand 

You should *not* do this. Run the `Anaconda Prompt` or `Anaconda Navigator` to launch `Anaconda Distribution` software, otherwise tick the checkbox to allow the installer to add the entries to PATH. It needs to add 4 entries.

Can you open a 'normal' cmd.exe prompt and type:

`echo %PATH%` and paste the results please?

**Comment by Rodolphe21** [comment]
here we go
[path.docx](https://github.com/ContinuumIO/anaconda-issues/files/1177892/path.docx)

**Comment by Rodolphe21** [comment]
bak to your advice on using "anaconda navigator": the application doesn't start and brings an error message "pythonw" not found.
> conda update anaconda-navigator
also doesn't work because it says it must install qt-6.5.2, something that doesn't work, see 4th issue reported above

And concerning the cmd window from the anaconda menu, see 2nd issue reported above...

Thank you for your help

Rodolphe

**Comment by mingwandroid** [comment]
Please paste the output as text, not as a MS Word document. I do not have MS Word installed.

**Comment by Rodolphe21** [comment]
sorry but I don't know how to do that in a cmd windows... do you have a command line for me?
Thank you

**Comment by mingwandroid** [comment]
You select all of the text with the mouse and then go to the top left of the cmd.exe window and go to Edit->Copy (or you may be able to just right click once with the mouse depending on your options), then paste it into a comment here with Control+V.

**Comment by Rodolphe21** [comment]
here we go...

Microsoft Windows [Version 6.1.7601]
Copyright (c) 2009 Microsoft Corporation.  All rights reserved.

C:\Users\rodolphe_masson>echo %PATH%

C:\ProgramData\Oracle\Java\javapath;C:\Windows\system32;C:\Windows;C:\Windows\Sy
stem32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\C
ommon Files\Roxio Shared\DLLShared\;C:\Program Files (x86)\Common Files\Roxio Sh
ared\10.0\DLLShared\;C:\Program Files (x86)\IVI Foundation\VISA\WinNT\Bin;C:\Pro
gram Files (x86)\Skype\Phone\;C:\Users\rodolphe_masson\AppData\Local\Continuum\A
naconda2\Scripts;C:\MinGW\bin;C:\Program Files (x86)\Microsoft VS Code\bin;C:\Us
ers\rodolphe_masson\AppData\Local\rodeo\app-2.4.10\bin

**Comment by mingwandroid** [comment]
I have no idea what's up with your system. Can you try the following:

1. Disable your virus scanner (temporarily).
2. From a normal cmd.exe prompt do:
```
cd %TEMP%
powershell -command "& { (New-Object Net.WebClient).DownloadFile('https://repo.continuum.io/archive/Anaconda2-4.4.0-Windows-x86_64.exe', 'ac3.exe') }"
start /wait "" ac3.exe /InstallationType=JustMe /AddToPath=0 /RegisterPython=0 /NoRegistry=1 /S /D=%TEMP%\ac3
%TEMP%\ac3\Scripts\activate.bat
python -c "from PyQt5 import QtWidgets"
echo %ErrorLevel% 0
```

.. and see if that works?

You mentioned installing matplotlib, that comes as part of Anaconda so you shouldn't need to install it. Also you mentioned an environment.yml file, can you paste the contents of that file?

**Comment by Rodolphe21** [comment]
Thank you very very much Mingwandroid for your great help.

Unfortunately, I cannot do what you are recommending, for (good) reasons I cannot detail here.
Let me spend today thinking about alternatives...


The content of the yml file is:

name: aind-dl
channels:
- defaults
dependencies:
- _nb_ext_conf=0.3.0=py35_0
- anaconda-client=1.6.2=py35_0
- bleach=1.5.0=py35_0
- bzip2=1.0.6=vc14_3
- clyent=1.2.2=py35_0
- colorama=0.3.7=py35_0
- cycler=0.10.0=py35_0
- decorator=4.0.11=py35_0
- entrypoints=0.2.2=py35_1
- freetype=2.5.5=vc14_2
- h5py=2.7.0=np112py35_0
- hdf5=1.8.15.1=vc14_4
- html5lib=0.999=py35_0
- icu=57.1=vc14_0
- ipykernel=4.5.2=py35_0
- ipython=5.3.0=py35_0
- ipython_genutils=0.1.0=py35_0
- ipywidgets=6.0.0=py35_0
- jinja2=2.9.5=py35_0
- jpeg=9b=vc14_0
- jsonschema=2.5.1=py35_0
- jupyter=1.0.0=py35_3
- jupyter_client=5.0.0=py35_0
- jupyter_console=5.1.0=py35_0
- jupyter_core=4.3.0=py35_0
- libpng=1.6.27=vc14_0
- libtiff=4.0.6=vc14_3
- markupsafe=0.23=py35_2
- matplotlib=2.0.0=np112py35_0
- mistune=0.7.4=py35_0
- mkl=2017.0.1=0
- nb_anacondacloud=1.2.0=py35_0
- nb_conda=2.0.0=py35_0
- nb_conda_kernels=2.0.0=py35_0
- nbconvert=5.1.1=py35_0
- nbformat=4.3.0=py35_0
- nbpresent=3.0.2=py35_0
- notebook=4.4.1=py35_0
- numpy=1.12.1=py35_0
- olefile=0.44=py35_0
- openssl=1.0.2k=vc14_0
- pandocfilters=1.4.1=py35_0
- path.py=10.1=py35_0
- pickleshare=0.7.4=py35_0
- pillow=4.0.0=py35_1
- pip=9.0.1=py35_1
- prompt_toolkit=1.0.13=py35_0
- pygments=2.2.0=py35_0
- pyparsing=2.1.4=py35_0
- pyqt=5.6.0=py35_2
- python=3.5.3=0
- python-dateutil=2.6.0=py35_0
- pytz=2016.10=py35_0
- pyyaml=3.12=py35_0
- pyzmq=16.0.2=py35_0
- qt=5.6.2=vc14_3
- qtconsole=4.2.1=py35_2
- requests=2.13.0=py35_0
- scikit-learn=0.18.1=np112py35_1
- scipy=0.19.0=np112py35_0
- setuptools=27.2.0=py35_1
- simplegeneric=0.8.1=py35_1
- sip=4.18=py35_0
- six=1.10.0=py35_0
- testpath=0.3=py35_0
- tk=8.5.18=vc14_0
- tornado=4.4.2=py35_0
- traitlets=4.3.2=py35_0
- vs2015_runtime=14.0.25123=0
- wcwidth=0.1.7=py35_0
- wheel=0.29.0=py35_0
- widgetsnbextension=2.0.0=py35_0
- win_unicode_console=0.5=py35_0
- zlib=1.2.8=vc14_3
- pip:
  - ipython-genutils==0.1.0
  - jupyter-client==5.0.0
  - jupyter-console==5.1.0
  - jupyter-core==4.3.0
  - keras==2.0.2
  - nb-anacondacloud==1.2.0
  - nb-conda==2.0.0
  - nb-conda-kernels==2.0.0
  - opencv-python==3.1.0.0
  - prompt-toolkit==1.0.13
  - protobuf==3.2.0
  - tensorflow==1.0.1
  - theano==0.9.0
  - tqdm==4.11.2
  - win-unicode-console==0.5

Thank you again!

Rodolphe

**Comment by mochapup** [comment]
Hello, 

I am having a very similar issue to Rodolphe
- Jupyter unable to update, Cannot open Anaconda Navigator, `conda update conda` updating to 4.3.24 and not 4.4, and qt update giving `defaults::qt-5.6.2-vc14_6'.
FileNotFoundError(2, 'The system cannot find the file specified', None, 2, None)` error. 

Has a solution been reached for this issue?

**Comment by Rodolphe21** [comment]
My solution was to move to another PC.
Couldn't really figure out whether the issue came from my Symantec Endpoint protection or from crowdstrike Falcon (on win7). Everything works well on my new PC (win10 with Mcafee)
Kind Regards,
Rodolphe

On 11. Aug 2017, at 22:09, Charles McDaniel <notifications@github.com<mailto:notifications@github.com>> wrote:


Hello,

I am having a very similar issue to Rodolphe

  *   Jupyter unable to update, Cannot open Anaconda Navigator, conda update conda updating to 4.3.24 and not 4.4, and qt update giving defaults::qt-5.6.2-vc14_6'. FileNotFoundError(2, 'The system cannot find the file specified', None, 2, None) error.

Has a solution been reached for this issue?
